.github/workflows/aggregate.yaml

```diff
@@ -1,54 +1,53 @@
 name: Aggregate daily data
 on:
-  schedule:
-    # at 6AM, 12AM, 6PM, and  12PM UTC
-    # at 2AM, 8AM, 2PM, and  8PM EST
-    - cron: "0 6/6 * * *"
-  workflow_dispatch:
+    schedule:
+        # every 30 minutes
+        - cron: '*/30 * * * *'
```

Comment:
So I think it's ok to do not consider the hour as we previously did (to consider the 50 untrusted blocks). We simply have to have that in mind at the time of running the token distribution